### PR TITLE
rpc, cli: allow to provide cosigners accounts to (*Client).CreateTxFromScript

### PR DESCRIPTION
**Pull request by AnnaShaleva**
We need cosigners' accounts to calculate the network fee properly.

### CODE REVIEW

File: cli/smartcontract/smart_contract.go [hunk 1]
```diff
@@ -554,10 +557,20 @@ func invokeInternal(ctx *cli.Context, signAndPush bool) error {
...
 		gas = flags.Fixed8FromContext(ctx, "gas")
-		acc, err = getAccFromContext(ctx)
+		acc, wall, err = getAccFromContext(ctx)
 		if err != nil {
 			return err
 		}
+		for i := range cosigners {
+			cosignerAcc := wall.GetAccount(cosigners[i].Account)
+			if cosignerAcc == nil {
+				return cli.NewExitError(fmt.Errorf("can't calculate network fee: no account was found in the wallet for cosigner #%d", i), 1)
```
Comment (fyrchik) [inline cli/smartcontract/smart_contract.go:567]: What happens if cosigner is a contract?

**Review by fyrchik — COMMENTED** [review]
Also some CLI tests are needed (we already have verify contract in testdata, test wallet may need to be generated).

File: cli/smartcontract/smart_contract.go (hunk 1 above)
Comment (AnnaShaleva) [reply cli/smartcontract/smart_contract.go:567]: It's OK for a cosigner to be a contract. `AddNetworkFee` is able to handle this case, it will call `verify` method for this contract to define network fee, like for any other contract signer.

But still, the cosigner (contract/sig/multisig) should be presented in the wallet (and have `isDeployed` == true for the contract cosigner). Otherwise, we won't be able to calculate the network fee for the transaction. 

I'll write  a test for it.

**Comment by AnnaShaleva** [comment]
> Also some CLI tests are needed (we already have verify contract in testdata, test wallet may need to be generated).

We already have `wallet1_solo.json` and it suits the test perfectly. Done tests.

File: cli/nep17_test.go [hunk 2]
```diff
@@ -83,6 +83,11 @@ func TestNEP17Balance(t *testing.T) {
 				e.checkNextLine(t, "^\\s*Updated\\s*:\\s*"+strconv.FormatUint(uint64(index), 10))
 			}
 		}
+
+		e.checkNextLine(t, "^\\s*$")
+		addr4, err := address.StringToUint160("NbxpLNCCSWZ9BkYpCYT8NfN1uoxq9Rfbrn") // deployed verify.go contract
+		require.NoError(t, err)
+		e.checkNextLine(t, "^Account "+address.Uint160ToString(addr4))
```
Comment (roman-khimov) [inline cli/nep17_test.go:90]: Unrelated?

**Comment by fyrchik** [comment]
Can you rebase this on master to see if test failure is gone?

File: cli/nep17_test.go (hunk 2 above)
Comment (AnnaShaleva) [reply cli/nep17_test.go:90]: Why? TestNEP17Balance will fail without it (because I deployed a new contract to the chain to test the changes).

File: cli/contract_test.go [hunk 3]
```diff
@@ -275,6 +277,81 @@ func TestComlileAndInvokeFunction(t *testing.T) {
 		})
 	})
 
+	t.Run("real invoke and save tx", func(t *testing.T) {
+		txout := path.Join(tmpDir, "test_contract_tx.json")
+		t.Cleanup(func() {
```
Comment (fyrchik) [inline cli/contract_test.go:282]: `tmpDir` is already removed in `Cleanup`

**Review by fyrchik — APPROVED** [review]

**Review by roman-khimov — APPROVED** [review]
